db.js

```diff
@@ -397,7 +399,7 @@ function DB(url) {
 // TODO: handle full urls, not just db names
 exports.use = function (url) {
     /* Force leading slash; make absolute path */
-    return new DB(url);
+    return new DB((url.substr(0, 1) !== '/' ? '/' : '') + url);
```

Comment:
This reverses the relative rewrite support added here?  https://github.com/kanso/db/blob/bc6755cc94771b4dfcf754a95ad0b6b5e3eba472/db.js